modules/ocf_kube/templates/etcd.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
```

Comment:
If this is templated by our convention it should end in .erb

Reply:
It would by the convention, but then you lose syntax highlighting on github and in your editor. I'm willing to change it, but I personally think it's fine as long as people know it's an erb template.

Reply:
imo it is not clear at all that it's an erb template outside of us in the know right now. You can configure syntax highlighting for different file extensions in your editor and there are cases where that will be wrong due to special characters that overlap between erb and the target language. YAML is pretty simple though so there might not be one in this case.

Reply:
ja++